src/js/core.js

```diff
@@ -899,7 +899,7 @@
 
             try {
                 this.events.disableCustomEvent('editableInput');
-                if (opts.url && opts.url.trim().length > 0) {
+                if (opts.url && opts.url.length > 0) {
```

Comment:
I missed this part "users can call this function externally". So I guess I need to add those 2 trim() calls back because otherwise it would allow users to add "empty strings" as url which is not correct.

I will update the PR.